src/ParseServer.js

```diff
@@ -110,6 +124,28 @@ class ParseServer {
     return this._app;
   }
 
+  async getSecurityChecks() {
+    const checks = await SecurityCheck.getChecks();
+    const logger = logging.getLogger();
+    const { Total } = checks;
+    delete checks.Total;
+    for (const category in checks) {
+      const data = checks[category];
+      for (const check of data) {
+        const { title, warning, error, result, success } = check;
+        if (result === 'success') {
+          logger.warn(`✅ ${success}\n`);
```

Comment:
All this logic should go into the security checks files, not into the server file.